### PR TITLE
Show upcoming disputes

### PR DESCRIPTION
**Pull request by nuevoalex**
[Clubhouse Story](https://app.clubhouse.io/augur/story/5860)

## Submitter checklist
- [X] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [X] Steps to reproduce
  - [X] Screenshot (If applicable)

## Reviewer Checklist
- [x] Test/lint pass 
- [ ] Post merge, story marked complete

### CODE REVIEW

File: src/modules/reporting/components/reporting-dispute/reporting-dispute.jsx [hunk 1]
```diff
@@ -15,7 +16,9 @@ export default class ReportingDispute extends Component {
     history: PropTypes.object.isRequired,
     doesUserHaveRep: PropTypes.bool.isRequired,
     markets: PropTypes.array.isRequired,
+    upcomingMarkets: PropTypes.array.isRequired,
     marketsCount: PropTypes.number.isRequired,
+    upcomingMarketsCount: PropTypes.number.isRequired,
```
Comment (justinbarry) [inline src/modules/reporting/components/reporting-dispute/reporting-dispute.jsx]: Why use the extra prop for length vs length checking within this file?

File: src/modules/reporting/components/reporting-dispute/reporting-dispute.jsx [hunk 2]
```diff
@@ -65,6 +68,25 @@ export default class ReportingDispute extends Component {
...
+                market={market}
+                disputeRound={p.disputeRound}
+                isMobile={p.isMobile}
+                location={p.location}
+                history={p.history}
+              />))
+        }
+        { p.upcomingMarketsCount === 0 &&
+          <NullStateMessage
+            message="No markets in this dispute window have been successfully disputed."
```
Comment (justinbarry) [inline src/modules/reporting/components/reporting-dispute/reporting-dispute.jsx]: Is this copy from the ticket? Was it discussed elsewhere?

File: src/modules/reporting/components/reporting-dispute/reporting-dispute.jsx (hunk 1 above)
Comment (nuevoalex) [reply src/modules/reporting/components/reporting-dispute/reporting-dispute.jsx]: following existing convention

File: src/modules/reporting/components/reporting-dispute/reporting-dispute.jsx (hunk 2 above)
Comment (nuevoalex) [reply src/modules/reporting/components/reporting-dispute/reporting-dispute.jsx]: Ah I got mine off of the mock screenshot, will change

File: test/reports/selectors/select-awaiting-dispute-markets-test.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
+import { describe, it } from 'mocha'
+import { assert } from 'chai'
+import proxyquire from 'proxyquire'
+
+describe(`modules/reports/selectors/select-awaiting-dispute-markets.js`, () => {
+  proxyquire.noPreserveCache().noCallThru()
+
+  const test = (t) => {
```
[the remark this answers is not in the document]
Comment (nuevoalex) [reply test/reports/selectors/select-awaiting-dispute-markets-test.js]: What part are you referring to specifically and how would you like it to be formatted instead?

**Pull request merged**